vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionAboutPageManager.java

```diff
@@ -63,4 +96,126 @@ public ExhibitionAboutPage getExhibitionAboutPage() {
         return aboutPageList != null && !aboutPageList.isEmpty() ? aboutPageList.get(0):new ExhibitionAboutPage();
     }
     
+    /**
+     * This method maps the title in ExhibitionAboutPage, and add that to
+     * exhibitionTitles list for each user selected Exhibition Language.
+    */
+    public void setAboutPageTitle(LocalizedTextForm title, ExhibitionAboutPage exhibitionAboutPage) {
```

Comment:
should be private or is that intended to be used anywhere outside of this class?